prow/config.yaml

```
@@ -64,6 +64,13 @@ deck:
       - podinfo.json
     - lens:
         name: buildlog
+        config:
+          highlight_regexes:
+            - "timed out"
+            - "ERROR:"
+            - "(FAIL|Failure \[)\b"
```

IIRC you need to escape `\` when using quoted strings. So this should be 
```suggestion
            - "(FAIL|Failure \\[)\\b"
```

---

Thanks! that is probably why the job failed too